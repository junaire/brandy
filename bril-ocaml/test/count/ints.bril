@main {
  a: int = const 2;
  b: int = const 3;
}
