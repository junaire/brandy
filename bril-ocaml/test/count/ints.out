Ints: 2 Bools: 0
